### ISSUE TITLE
emacs doesn't compile on samsung chromebook 3

### ISSUE THREAD

**Issue by cstrouse** [skycocker/chromebrew#271]
On the Samsung Chromebook 3 (XE500C13-K01US) emacs fails to compile due to missing selinux.h. Is the best approach to just add --without-selinux to the package for everyone or should this be dealt with conditionally by checking for selinux first? Once we have consensus on the best approach I'd be more than happy to submit a PR with a fix.

```
Version 55.0.2883.87 (64-bit)
Platform 8872.70.0 (Official Build) stable-channel celes
ARC Version 3554338
Firmware Google_Celes.7287.92.60
```

**Comment by thedamian** [comment]
I guess we don't have dependancy requirements per platform 

I'm a little blown away that something is missing from the arm platform.

My preference is  nano but emacs is very important.

If we make a selinux.rb package is there a way to have it do nothing on x86 devices?

**Comment by bfayers** [comment]
XE500C13s have celerons, as far as I can find @thedamian

**Comment by lyxell** [comment]
Correct, XE500C13-K01US is x86_64.

**Comment by thedamian** [comment]
Alright. Is there anyone out there with an atom 32bit or celeron 32bit version that can install emacs successfully (if yes, then we have nothing to do here my friends) 

otherwise we can change the requirements for 32bit only since it'll affect them only.

**Comment by thedamian** [comment]
It compiled. And installed fully and tested on a deletion 32bit Chromebook (the original CR-48)

So I'm feeling we have to close this issue and it may have to do with your system.

**Comment by cstrouse** [comment]
I modified my local copy of the emacs package to disable selinux support since the headers aren't present on the 64-bit version. I guess close it and if someone else hits this same snag we can revisit it. Thanks for looking into this @thedamian.

**Comment by lyxell** [comment]
It would be interesting to know if it works on other 64-bit boards, like the Chromebook Pixel.

**Comment by cstrouse** [comment]
Shall I just add --disable-selinux to emacs then as well?

**Comment by lyxell** [comment]
@cstrouse That's a reasonable workaround in my opinion. At least until we have a better solution.

**Comment by cstrouse** [comment]
Resolved with #305.
